libs/wingsdk/src/std/array.ts

```diff
@@ -236,12 +236,15 @@ export class MutArray {
   }
 
   /**
-   * Add value to end of array
-   * @param value value to add
+   * Add values to end of array
+   *
+   * @macro ((obj, args) => { obj.push(...args); })($self$, [$args$])
+   *
+   * @param values values to add
    */
-  public push(value: T1): void {
-    value;
-    throw new Error("Abstract");
+  public push(...values: T1[]): void {
```

Comment:
Please add a wing test [here](https://github.com/autow0rk/wing/blob/attach-updated-arrayTS-file-5114/examples/tests/sdk_tests/std/array.test.w#L75).
No need to update the snapshots manually - once you commit your changes the mutation job will update the snapshots for you.

Reply:
Added the wing test. Should I remove the manually updated snapshots to let the mutation job run or just keep it?

Reply:
Ok to just keep it